README.md

```diff
@@ -24,13 +24,24 @@ We should only use static HTML and CSS for this project. Preprocessors (haml, er
 
 This group fansite must have the following HTML files:
 
-- `index.html` must include your names or team name somewhere.
+- `index.html` must include your names or team name somewhere. This file is in the project root.
 
-- `facts.html` should include a series of fun facts or more detailed information about your topic.
+- `facts.html` should include a series of fun facts or more detailed information about your topic. This file is in the `pages` folder.
 
-- `gallery.html` must include at least 3 images with descriptions and links to where the images came from if applicable. 
+- `gallery.html` must include at least 3 images with descriptions and links to where the images came from if applicable. This file is also in the `pages` folder.
 
-Each group member should take responsibility for creating one HTML file. 
+Each group member should take responsibility for completing one HTML file.
+
+##### For Groups With More Than 3 Members
+
+Add an additional page of your choice for each additional group member!
+
+Some possibilities for additional pages could include
+- a testimonials or reviews page
+- a frequently asked questions page
+- a blog entry (such as described in the [Optional Enhancements](#optional-enhancements) below)
+
+Any extra pages you add are limited only by your imagination, but they should involve a comparable amount of work and conform to all of the same project requirements as any of the required pages.
```

Comment:
Maybe me being over-anxious, but the original wording left room in my brain that these additional pages may not be required for a team of 4 or more since here "required pages" is only referring to the initial 3 above (if that explanation makes any sense).

```suggestion
Any extra pages you add are limited only by your imagination, but they should involve a comparable amount of work and conform to all of the same project requirements as any of the required pages in the section above.
```